config/job_types/ingest_journal.yml

```
@@ -43,14 +44,14 @@ tasks:
     foreach:
       list_files: jpg
       foreach:
-        task: image.scaling
+        task: convert.image_scaling
```

```suggestion
        task: convert.scale_image
```
Unsere tasks sind in der Regel im Imperativ formuliert.

---

Ich merge gerade die Änderungen hier mit meinen aus diser Woche. Ich ändere das entsprechend deines Vorschlags.